src/blocks/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 406px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.picture {
+  position: absolute;
+  width: 160px;
+  height: 134px;
+  left: 28px;
+  top: 41px;
+}
+
+.title {
+  margin: 191px 16px 2px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.code {
+  font-style: normal;
+  font-weight: 350;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  padding: 0 16px;
+  margin-top: 2px;
+  margin-bottom: 7px;
+}
+
+.reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 16px;
+  margin-bottom: 14px;
+}
+
+.reviews__number {
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.price {
+  margin-bottom: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 16px;
+}
+
+.price__letters {
+  margin: 0;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 16px;
+  color: #616070;
+}
+
+.price__numbers {
+  margin: 0;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.buy-bottom {
+  margin: 0 16px;
+  height: 38px;
+  width: 164px;
```

Comment:
do not hardcode width and height
width should be 100%
use paddings to set height